packages/features/bookings/lib/handleCancelBooking.ts

```diff
@@ -151,7 +152,7 @@ async function handler(req: NextApiRequest & { userId?: number }) {
       ? parseRecurringEvent(bookingToDelete.eventType?.recurringEvent)
       : undefined,
     location: bookingToDelete?.location,
-    destinationCalendar: bookingToDelete?.destinationCalendar || bookingToDelete?.user.destinationCalendar,
+    destinationCalendar: bookingToDelete?.destinationCalendar || bookingToDelete?.user[0].destinationCalendar,
```

Comment:
If there is no destination calendar for that booking, then default to the first user's destination calendar